tests/test_api.py

```diff
@@ -947,6 +947,12 @@ def test_orchestrator_build_logs_api_badlog(self, osbs):  # noqa
         assert isinstance(content, six.string_types)
         assert content == u"   líne 1"
 
+    def test_orchestrator_build_logs_no_logs(self, osbs):  # noqa
```

Comment:
You can use `# noqa:F811` to avoid a diagnostic about `osbs` being redefined. That way we'll still see any other diagnostics there might be. (I know other methods in that file also use `# noqa` without any specific diagnostic being named... they ought to be corrected too, but that's outside the scope of this PR.)

Reply:
done